packages/api-explorer/src/components/SideNav/SideNavMethods.tsx

```diff
@@ -40,33 +49,111 @@ interface MethodsProps {
   specKey: string
 }
 
+const SideNavDisclosure = styled(AccordionDisclosure)<{ isOpen: boolean }>`
+  padding-left: ${({ theme }) => theme.space.large};
+  padding-right: ${({ theme }) => theme.space.large};
+
+  color: ${(props) =>
+    props.isOpen ? ({ theme }) => theme.colors.key : 'inherit'};
+
+  ${Icon} {
+    color: ${(props) =>
+      props.isOpen ? ({ theme }) => theme.colors.key : 'inherit'};
+  }
+
+  &:hover,
+  &:focus {
+    color: ${({ theme }) => theme.colors.key};
+  }
+`
+
+const SideNavContent = styled.div`
+  padding: ${({
+    theme: {
+      space: { xxsmall, large },
+    },
+  }) => `${xxsmall} ${large}`};
+`
+
+const SideNavList = styled(List)`
+  border-left: dashed 1px ${({ theme }) => theme.colors.ui2};
+  padding: ${({ theme }) => theme.space.xxsmall} 0
+    ${({ theme }) => theme.space.xxsmall} ${({ theme }) => theme.space.xxsmall};
+`
+
+const SideNavListItem = styled(ListItem)`
+  margin-bottom: 0;
+
+  a {
+    border-radius: ${({ theme: { radii } }) => radii.medium};
+    cursor: pointer;
+    display: block;
+    padding: ${({ theme }) => theme.space.xsmall};
+
+    &:hover,
+    &:focus,
+    &.active {
+      background-color: ${({ theme }) => theme.colors.ui1};
+
+      ${MethodBadge} {
+        border: solid 1px ${({ theme }) => theme.colors.ui2};
+      }
+    }
+
+    &.active {
+      ${ApixHeading} {
+        font-weight: ${({ theme }) => theme.fontWeights.semiBold};
+      }
+    }
+  }
+`
+
```

Comment:
These should live in `ExplorerStyle.tsx`

Reply:
@josephaxisa - There was some functionality here like `isOpen` that I wasn't sure how to move into ExplorerStyle.tsx if you can give me clarity on what I would need to bring over into ExplorerStyle.tsx to keep everything from breaking I am happy to clean this up further.